zorg/buildbot/builders/FlangBuilder.py

```diff
@@ -19,7 +19,7 @@ def getFlangOutOfTreeBuildFactory(
     f = getCmakeWithNinjaBuildFactory(
             depends_on_projects=['llvm','clang','mlir','openmp','flang','flang-rt'],
             enable_projects=['llvm','clang','mlir'],
-            enable_runtimes=['openmp'],
+            enable_runtimes=['openmp', 'compiler-rt'],
```

Comment:
I'm not sure, but I think that 'compiler-rt' should also be added to `depends_on_projects`.
My understanding is that this way builds are also triggered by changes in compiler-rt.